apps/examples/testcase/le_tc/kernel/Kconfig

```diff
@@ -207,7 +208,9 @@ config TC_KERNEL_TASH_HEAPINFO
 config TC_KERNEL_TASH_STACKMONITOR
 	bool "StackMonitor"
 	default n
-
+config TC_KERNEL_WORK_QUEUE
```

Comment:
Could you make a empty line before and after to sparate different config?

Reply:
Okay